# Checks imports for absolute paths. (no-absolute)

Disallow all absolute imports (starting with a /).


## Rule Details

Examples of **incorrect** code for this rule:

```js

import '/some/local/path';

```
